can-bind.js

```diff
@@ -380,6 +380,25 @@ canAssign(Bind.prototype, {
 				}
 			}
 
+			//!steal-remove-start
+			if(process.env.NODE_ENV !== 'production'){
+				// Here we want to do a dev-mode check to see whether the child does type conversions on
+				//  any two-way bindings.  This will be ignored and the child and parent will be desynched.
+				var childContext = options.child.observation && options.child.observation.func || options.child;
+				parentValue = canReflect.getValue(options.parent);
+				childValue = canReflect.getValue(options.child);
+				if (parentValue != null && childValue !== parentValue) {
```

Comment:
`!= null` .. should this be `!== undefined`?  Why is there a nullish check?  One can have a parent to child binding where the parent is initially undefined.  One can have a child to parent where the child is initially undefined.

Reply:
I've addressed the above concerns.  Check out the new version.